### DISCUSSION TITLE
DB: failed to download vulnerability

### DISCUSSION THREAD

**Discussion by northboys** [aquasecurity/trivy#1601] [Q&A] [answered]
i tried to use trivy in gitlab-ci and got error: 
`FATAL	DB error: failed to download vulnerability DB: failed to download vulnerability DB: failed to list releases: GET https://api.github.com/repos/aquasecurity/trivy-db/releases: 401 Bad credentials []`

How to fix it?

**Comment by knqyf263** [comment, marked as answer]
You may set `GITHUB_TOKEN`. If it is the case, could you unset that and try again?
